kolibri/core/assets/src/exams/utils.js

```diff
@@ -102,7 +102,13 @@ function getExamReport(store, examId, userId, questionNumber = 0, interactionInd
             const itemId = currentQuestion.itemId;
             const exercise = contentNodeMap[currentQuestion.contentId];
             const currentAttempt = allQuestions[questionNumber];
-            const currentInteractionHistory = currentAttempt.interaction_history;
+            // filter out interactions without answers but keep hints and errors
+            const currentInteractionHistory = currentAttempt.interaction_history.filter(
+              interaction =>
+                Boolean(
+                  interaction.answer || interaction.type === 'hint' || interaction.type === 'error'
```

Comment:
Shouldn't be showing hints on exams - but in case we want to make them more formative friendly by calling them quizzes, I think this is fine to leave in!